### PR TITLE
[planet-finance-v2] Planet V2 voting strategy

### PR DESCRIPTION
**Pull request by defininja**
Fixes # .

- Planet Finance updated voting strategy to give 5X voting power for AQUA infinity vault stakers

### CODE REVIEW

**Comment by defininja** [comment]
@ChaituVR - raised a new pull request keeping the old strategy and creating new v2 strategy. Request you to approve it. Thanks.

File: src/strategies/planet-finance-v2/index.ts [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    { blockTag }
+  );
+
+  const result = await Promise.all([
+    score,
+    usersAquaVaultBalances,
+    usergAquaBalInAquaInfinityVault,
+    usersAquaGammaVaultBalances,
+    usersNewAquaBnbVaultBalances,
+    usersAquaInLending
```
Comment (ChaituVR) [inline src/strategies/planet-finance-v2/index.ts:139]: Hi @defininja There is limit of number of number of requests/multicall made from a strategy,


Is there some way to reduce number of calls being sent? and for erc20BalanceOfStrategy you can use `erc20-balance-of` strategy in your space, resulting voting power will be sum of these strategies

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/planet-finance-v2/index.ts (hunk 1 above)
Comment (defininja) [reply src/strategies/planet-finance-v2/index.ts:139]: @ChaituVR - what is the limit on the number of requests and multicalls? Can you please elaborate?

Comment (defininja) [reply src/strategies/planet-finance-v2/index.ts]: Hi @ChaituVR,

I have updated code to match your requirements. Please approve my pull request.
Thanks.

https://github.com/defininja/snapshot-strategies/tree/b380c66d8a87e4879e7b74e108e18bf5e5404fd9

Comment (defininja) [reply src/strategies/planet-finance-v2/index.ts]: hi @ChaituVR - can you please look into it? Thanks.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**